src/utils/web-utils.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export function wait(waitTimeMs: number): Promise<void> {
```

Comment:
nit: We usually capitalize the file name so should be Web and does not use dash. So should be WebUtils.ts
Also should we just call it Utils.ts

Reply:
As discussed, I have updated the file name and corresponding changes. Resolving this now, thanks!